vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePickerVariant.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2000-2018 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datepicker;
+
+/**
+ * Set of theme variants applicable for {@code vaadin-date-picker} component.
+ */
+public enum DatePickerVariant {
+    LUMO_SMALL("small"), LUMO_ALIGN_CENTER("align-center"), LUMO_ALIGN_RIGHT(
+            "align-right"), LUMO_HELPER_ABOVE_FIELD("helper-above-field");
```

Comment:
Is there any reference for a comprehensive list of applicable themes? To check this I cross-referenced several style modules in the web-components repo, but I'm not sure if I found all of them.

One theme that is missing is `align-left`. I assume that it was left out intentionally since it seems to be the default. We should still add it to have the complete set. It might also be required when using RTL, and you want to align he text left for some reason.

Reply:
> One theme that is missing is align-left. I assume that it was left out intentionally since it seems to be the default. 

Yes, I just followed how it was done in TextField.